fs/config/config.go

```diff
@@ -53,15 +53,16 @@ const (
 )
 
 type Config struct {
-	HTTPCacheType       string `toml:"http_cache_type"`
-	FSCacheType         string `toml:"filesystem_cache_type"`
-	ResolveResultEntry  int    `toml:"resolve_result_entry"`
-	NoBackgroundFetch   bool   `toml:"no_background_fetch"`
-	Debug               bool   `toml:"debug"`
-	AllowNoVerification bool   `toml:"allow_no_verification"`
-	DisableVerification bool   `toml:"disable_verification"`
-	MaxConcurrency      int64  `toml:"max_concurrency"`
-	NoPrometheus        bool   `toml:"no_prometheus"`
+	HTTPCacheType                    string `toml:"http_cache_type"`
+	FSCacheType                      string `toml:"filesystem_cache_type"`
+	ResolveResultEntry               int    `toml:"resolve_result_entry"`
+	NoBackgroundFetch                bool   `toml:"no_background_fetch"`
+	Debug                            bool   `toml:"debug"`
+	AllowNoVerification              bool   `toml:"allow_no_verification"`
+	DisableVerification              bool   `toml:"disable_verification"`
+	MaxConcurrency                   int64  `toml:"max_concurrency"`
+	NoPrometheus                     bool   `toml:"no_prometheus"`
+	BackgroundFetchSilencePeriodMSec int    `toml:"background_fetch_silence_period_msec"`
```

Comment:
If we just specified all our configurable time intervals in milliseconds, we wouldn't have to have this awkward naming. But okay.

Also, task manager calls this `prioritizedTaskSilencePeriod.` I think I prefer that name, because it's clear what we're being silent in relation to. The trouble with `BackgroundFetchSilencePeriod`, is that someone might think it's a startup thing that prevents background tasks from running until X seconds post-launch. We do need such a config parameter as well. What would you call that, that would clearly disambiguate it from this?

Reply:
Renamed to `PrioritizedTaskSilencePeriodMSec`.